components/alerts/src/css/simple-alert.css

```diff
@@ -0,0 +1,227 @@
+@import url(../../../../css/component.css);
+.warning-button {
```

Comment:
Maybe you can refactor this to have a common .button class with all the common property under one class.

Reply:
I am trying to create different buttons as of now and will generalize once I finalize the concept of ecommerce website.
Thanks for your valuable input Humza.